distribution/changelog.txt

```diff
@@ -22,6 +22,7 @@
 - Fix: Incomplete loop collision box allowed overlapping track (original bug).
 - Improved: [#12806] Add Esperanto diacritics to the sprite font.
 - Improved: [#12890] Add stroke to lowercase 'L' to differentiate from capital 'I'.
+- Improved: Changed peep movement so that they stay more spread out over the full width of paths.
```

Comment:
This would imply they use the full width of a multitile path but you mean the full width of a single path tile.

Reply:
```suggestion
- Improved: Changed peep movement so that they stay more spread out over the full width of single tile paths.
```